Composer/packages/lib/code-editor/src/LuEditor.tsx

```diff
@@ -239,7 +241,7 @@ const LuEditor: React.FC<LULSPEditorProps> = (props) => {
           if (luEdits?.scrollLine) {
             editor.revealLineInCenter(luEdits?.scrollLine);
           }
-
+          telemetryClient.track('LUEditorEntityDefinitionAdded', { entityType });
```

Comment:
I think ```LUEditorToolbar``` should be the prefix for all the toolbar events, the keyword is toolbar